upi/vsphere/install-config.yaml.example

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
nit: no need for this leading document separator, and most of our other YAML examples omit it.